### PR TITLE
CORE-1360 Add terminate dlg for analyses

### PR DESCRIPTION
**Pull request by ash3rz**
From our last discussion on this ticket, VICE analyses will be terminated and marked as Completed, while non-VICE analyses will be terminated and marked as Canceled.

image

image

### CODE REVIEW

File: src/components/analyses/toolbar/TerminateAnalysisDialog.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    const analysisCount = selectedAnalyses?.length;
+
+    const handleTerminateConfirmed = () => {
+        const mapIsVice = groupBy(selectedAnalyses, isInteractive);
+        const viceAnalyses = mapIsVice?.true;
+        const condorAnalyses = mapIsVice?.false;
+
+        handleSaveAndComplete(viceAnalyses);
+        handleCancel(condorAnalyses);
+    };
```
Comment (psarando) [inline src/components/analyses/toolbar/TerminateAnalysisDialog.js:40]: 👍

**Review by psarando — APPROVED** [review]
LGTM :shipit: 👍

**Review by sri2k1us — APPROVED** [review]
:shipit:

**Comment by ash3rz** [comment]
Thanks for the reviews!

**Pull request merged**